FlowCrypt/Controllers/Compose/ComposeViewController.swift

```diff
@@ -1085,8 +1087,11 @@ extension ComposeViewController {
         }
     }
 
-    private func handleEvaluation(error: Error, with email: String) {
+    private func handleEvaluation(error: Error, with email: String, localContact: RecipientWithSortedPubKeys?) {
         let recipientState: RecipientState = {
+            if let _localContact = localContact {
```

Comment:
Here let's only do this if the local contact has a valid state. There could be other states like expired, revoked.

If the local contact is expired, and the lookup fails, I want the result of the lookup to fail visibly, so that the user knows to retry (since the public key that is remote could be not-expired - it's worth retrying).

If the local pubkey is already valid, then there is no need for retrying the remote fetch and that's why we can ignore the fetch error and render it.